airbyte-integrations/connectors/destination-postgres/src/integrationTestLegacy/kotlin/io/airbyte/integrations/destination/postgres/typing_deduping/PostgresRawOverrideDisableTypingDedupingTest.kt

```diff
@@ -17,11 +19,31 @@ class PostgresRawOverrideDisableTypingDedupingTest : PostgresTypingDedupingTest(
     override val rawSchema: String
         get() = "overridden_raw_dataset"
 
+    // when disable_type_dedupe = true, we only output to raw table
     override fun disableFinalTableComparison(): Boolean {
         return true
     }
 
+    override fun disableRawTableComparison(): Boolean {
+        return false
+    }
+
     @Disabled @Test override fun identicalNameSimultaneousSync() {}
 
     @Disabled @Test override fun testVarcharLimitOver64K() {}
+
+    // this test assumes that fields that are not in the schema will show up
+    // on the raw table. This is only true for older versions of the connector.
+    @Disabled @ParameterizedTest @ValueSource(longs = []) override fun testIncrementalSyncDropOneColumn(inputGenerationId: Long) {}
+
+    // this test assumes that the fully qualified raw table name is lowercased.
+    // This was only a restriction in older versions of the connector.
+    @Disabled @Test override fun testMixedCasedSchema() {}
+
+    // disabling dedup tests since dedup not supported when setting `disable_type_dedupe` to true.
```

Comment:
Note that even though running dedup when disable_type_dedupe is set to true doesn’t make a ton of sense, it is a config that wouldn’t error out in older versions of the connector, but this will error out on the new version of the connector.